lib/pry/config/behavior.rb

```diff
@@ -1,6 +1,41 @@
 class Pry
   class Config < Pry::BasicObject
-    # rubocop:disable Metrics/ModuleLength
+    #
+    # {Pry::Config::Behavior} is a module who can be included by classes who
+    # wish to behave similar to an OpenStruct object:
+    #
+    # ```ruby
+    # class Store
+    #   include Pry::Config::Behavior
```

Comment:
1. Is this class a public API? If yes, then why?
2. Who else includes this module besides 1 class in Pry?

Reply:
1. I doubt anyone has used this module directly, but sure if someone wants to, i suppose they could. 

2. `Pry::Config` and `Pry::Config::Default` both include it, although https://github.com/pry/pry/pull/1897 changes that to just `Pry::Config`.

Reply:
> We should mark it as private, then. Currently, it's shown as public.

Don't feel that strongly one way or the other, i don't think it will make a difference.

> Why to have this module at all? Only one class will use it. It's easier to get this inlined rather than move it to a separate file.

Because it allows for other classes to inherit this behaviour without enforcing itself as the superclass, and keeps `Pry::Config` tidy and slim.

I don't see why we would remove that flexibility, especially for what seems like a "just because".